src/components/Inputs/SimpleInput/SimpleInput.tsx

```diff
@@ -75,6 +77,24 @@ export const AmountInput: React.FC<IProps> = ({
       disableUnderline={true}
       placeholder={placeholder}
       onChange={allowOnlyDigitsAndTrimUnnecessaryZeros}
+      endAdornment={
+        globalPrice ? (
+          <Tooltip
+            title={
+              <Typography className={classes.textGlobalPrice} variant='caption'>
+                Global price
+              </Typography>
+            }
+            placement='right-start'
+            classes={{
+              tooltip: classes.globalPriceTooltip
+            }}>
+            <Typography className={classes.globalPrice} variant='h4'>
+              {globalPrice > 1 ? globalPrice.toFixed(2) : globalPrice}
```

Comment:
Change precision to `0.0001. This accuracy is not only intended for prices below 1. Consider, for example, the case of stablecoins: The ratio is displayed with high precision only in one direction, upon inrversal, the ratio is already rounded to `10^(-2)`

Reply:
fix applied